pkg/gits/git_cli.go

```diff
@@ -868,6 +873,12 @@ func (g *GitCLI) GetLatestCommitMessage(dir string) (string, error) {
 	return g.gitCmdWithOutput(dir, "log", "-1", "--pretty=%B")
 }
 
+// GetLatestTag returns the latest git commit message
+func (g *GitCLI) GetLatestTag(dir string) (string, error) {
```

Comment:
How does this differ from `NthTag(<dir>, 1)`? I am a bit concerned about how `GitCLI` is developing. I am not a big fan of `NthTag` either, but since it's there we might as well use it. @macox WDYT?